Modules/ThirdParty/VNL/src/CMakeLists.txt

```diff
@@ -20,6 +20,9 @@ set(BUILD_EXAMPLES ${BUILD_EXAMPLES} CACHE BOOL "Build the examples from the ITK
 
 foreach(lib itkvcl itkv3p_netlib itktestlib itkvnl itkvnl_algo itknetlib)
   itk_module_target(${lib} NO_INSTALL)
+  if (MSVC)
```

Comment:
Here we want to condition on the Visual Studio generator instead of `MSVC` in case we are using ninja.

Reply:
As `ninja` is not using `MSBuild`, I think this option will be irrelevant. Also, `ninja` is always using object-level parallelism. And I am compiling this using `ninja` as we speak, and the build is looking normally with 100% CPU use.